src/observations/test_price_observation.py

```
@@ -0,0 +1,77 @@
+import pytest
+import numpy as np
+import pandas as pd
+from src.observations.price_observation import PriceObservation, PriceEnvironment
```

Ruff

`src.observations.price_observation.PriceEnvironment` imported but unused

Remove unused import: `src.observations.price_observation.PriceEnvironment`

(F401)